lib/connections/service.go

```diff
@@ -130,6 +130,7 @@ type service struct {
 	connectionStatusHandler
 
 	cfg                  config.Wrapper
+	configDevicesUpdated chan bool
```

Comment:
Feel free to ignore this, but it could be a zero sized `struct{}`.

Reply:
Equally ignorable (cause bike-shedding): `deviceAddressesChanged` would be more accurate in my opinion.